prov/psm/src/psmx_rma.c

```diff
@@ -752,7 +752,8 @@ ssize_t _psmx_write(struct fid_ep *ep, const void *buf, size_t len,
 			nargs++;
 		}
 
-		if (flags & FI_REMOTE_COMPLETE) {
+		/* TODO: unsure how to make FI_TRANSMIT_COMPLETE the default */
+		if (flags & FI_TRANSMIT_COMPLETE) {
```

Comment:
The flag to check here is actually FI_COMMIT_COMPLETE.  The default behavior when flags is 0 does satisfy the transmit complete semantics.